src/crypto/iota_crypto.c

```diff
@@ -120,6 +120,17 @@ int iota_crypto_sign(uint8_t const priv_key[], uint8_t msg[], size_t msg_len, ui
 #endif
 }
 
+int iota_crypto_sign_open(uint8_t msg[], size_t msg_len, uint8_t const pub_key[], uint8_t signature[]) {
+#if defined(CRYPTO_USE_SODIUM)
+  unsigned long long sign_len = ED_SIGNATURE_BYTES;
+  return crypto_sign_ed25519_open(msg, (unsigned long long *)&msg_len, signature, sign_len, pub_key);
```

Comment:
`sign_len` variable is redundant.

```suggestion
  return crypto_sign_ed25519_open(msg, (unsigned long long *)&msg_len, signature, (unsigned long long)ED_SIGNATURE_BYTES, pub_key);
```